test/rsspp_parser.cpp

```diff
@@ -284,10 +295,11 @@ TEST_CASE("Extracts data from media:... tags in  RSS 2.0 feeds",
 	REQUIRE(f.items[0].description_mime_type == "text/html");
 	REQUIRE(f.items[0].link == "http://example.com/player.html");
 
-	REQUIRE(f.items[1].title ==
-		"using multiple media tags nested in group/content");
```

Comment:
Why did we stop testing the title?

Reply:
I moved the title fallback functionality (using media:title if no regular title is specified) into src/rssparser.cpp.
However, while looking into this, I'm finding several (small) issues, I'm thinking about fixing those first in a separate PR.
Either way, I've added back the test step (now checking for empty title) and added a test for `RssReader`.

Reply:
My original idea was to make the `rsspp::{Feed,Item}` classes be a direct representation of the source rss/atom data, and only make decisions about fallbacks and conversions in `newsboat::RssParser`. That might still be nice to do, but while rebasing, I realized that was not at all necessary for this PR (and I think it was making some changes in the fallback order for article content)

I rolled back the change for now, I might look into this in a later PR.